docs/geps/gep-04.rst

```diff
@@ -0,0 +1,185 @@
+=========================================
+GEP 4 — A DAG—based Computational Backend
+=========================================
+
++------------+---------------------------------------------------------------+
+| Author     | `Max Blesch <https://github.com/MaxBlesch>`_                  |
++            +---------------------------------------------------------------+
+|            | `Janos Gabler <https://github.com/janosg>`_                   |
++            +---------------------------------------------------------------+
+|            | `Hans-Martin von Gaudecker <https://github.com/hmgaudecker>`_ |
++            +---------------------------------------------------------------+
+|            | `Tobias Raabe <https://github.com/tobiasraabe>`_              |
++------------+---------------------------------------------------------------+
+| Status     | Draft                                                         |
++------------+---------------------------------------------------------------+
+| Type       | Standards Track                                               |
++------------+---------------------------------------------------------------+
+| Created    | 2020-05-05                                                    |
++------------+---------------------------------------------------------------+
+| Resolution | <url> (required for Accepted or Rejected or Withdrawn)        |
++------------+---------------------------------------------------------------+
+
+
+Abstract
+--------
+
+This GEP explains the DAG-based computational backend for gettsim which does not only
+increase performance, but, and more importantly, offers a way to make small changes to
```

Comment:
"small changes" sounds like you cannot use gettsim to model radically different policies. You can make small or big changes to the tax system but the required change in code is almost as small as it can be.

Reply:
Fixed.